dotcom-rendering/src/client/islands/doHydration.tsx

```diff
@@ -29,24 +28,23 @@ export const doHydration = async (
 	const alreadyHydrated = element.dataset.guReady;
 	if (alreadyHydrated) return;
 
-	const { start: importStart, end: importEnd } = measureDuration(
-		`import-${name}`,
+	const { endPerformanceMeasure: endImport } = startPerformanceMeasure(
```

Comment:
what do you think about this?

`endImport` reads a bit like it will stop the import from finishing (appreciate it's very similar to current code)

```suggestion
	const { endPerformanceMeasure: endImportPerformanceMeasure } = startPerformanceMeasure(
```